### PR TITLE
docs: ui.contextual_help

### LINKED ISSUE #832: docs: ui.contextual_help

### PR DESCRIPTION
**Pull request by AkshatJawne**
Closes #832

### CODE REVIEW

**Review by dsmmcken — CHANGES_REQUESTED** [review]

File: plugins/ui/docs/components/contextual_help.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Contextual Help
+
+The contextual help feature shows extra information about the state of a component, or a total view.
```
[the remark this answers is not in the document]
Comment (AkshatJawne) [reply plugins/ui/docs/components/contextual_help.md:3]: I had meant view as in an intentional a grouping of components, sort of related to ui.view. But yeah, getting rid of this, not super helpful.

**Review by dsmmcken — CHANGES_REQUESTED** [review]

**Review by dsmmcken — APPROVED** [review]

File: plugins/ui/docs/components/contextual_help.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    heading="Need Help",
+    content="If you are having issues accessing your account, contact our customer support team for help.",
+    variant="info",
+)
+```
+
+
+## Placement
+
+The contextual help component supports different placement options for when the positioning of the popover needs to customized.
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/contextual_help.md:23]:
```suggestion
The contextual help component supports different placement options for when the positioning of the popover needs to be customized.
```

File: plugins/ui/docs/components/contextual_help.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    ]
+
+
+my_contextual_help_placement_examples = ui_contextual_help_placement_examples()
+```
+
+
+## Events
+
+The contextual help componenthas an `on_open_change` prop, triggered when the the popover opens or closes.
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/contextual_help.md:58]:
```suggestion
The `on_open_change` prop is triggered when the popover opens or closes.
```

or at least: The contextual help component has an `on_open_change` prop, triggered when the popover opens or closes.

**Review by dsmmcken — DISMISSED** [review]

File: plugins/ui/docs/components/contextual_help.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    heading="Need Help",
+    content="If you are having issues accessing your account, contact our customer support team for help.",
+    variant="info",
+)
+```
+
+
+## Placement
+
+The contextual help component supports different placement options for when the positioning of the popover needs to be customized.
```
Comment (margaretkennedy) [inline plugins/ui/docs/components/contextual_help.md:23]:
```suggestion
The contextual help component supports different placement options for when the popover's positioning needs to be customized.
```

**Review by margaretkennedy — APPROVED** [review]

**Pull request merged**